src/runtime/sysvar/slot_history.zig

```diff
@@ -22,8 +24,19 @@ pub const SlotHistory = struct {
     pub const ID =
         Pubkey.parseBase58String("SysvarS1otHistory11111111111111111111111111") catch unreachable;
 
-    pub fn deinit(self: SlotHistory, allocator: std.mem.Allocator) void {
-        sig.bincode.free(allocator, self);
+    pub const SIZE_OF: u64 = 131_097;
+
+    pub fn default(allocator: Allocator) Allocator.Error!SlotHistory {
+        var bits = try DynamicArrayBitSet(u64).initEmpty(allocator, MAX_ENTRIES);
+        bits.set(0);
```

Comment:
why is this the default?

Reply:
Maybe the genesis slot should explicitly be added by the user when processing slots starting from genesis. What do you think? I don't have enough context to have a preference. I trust your judgment